docs/adrs/0020-pipeline-academies-from-adb.md

```diff
@@ -0,0 +1,108 @@
+- **Complete**
+
+Long term, the optimal solution is to consume this data through the APIs provided by each respective application. However, the timelines for implementing this feature make it impractical to expect these projects to deliver their APIs in time. 
+
+To meet our deadlines, we propose an interim approach: leveraging the existing synchronisation mechanisms where these applications sync their data back to the Academies Database daily for reporting purposes. This allows us to utilise a consolidated and up-to-date data source for our needs while maintaining acceptable performance and reliability.
+
+## Decision
+
+For the interim, we will integrate data from the Academies Database (ADB) to support the "Pipeline Academies" feature. This approach involves querying specific tables within ADB, which aggregate data synced from the relevant applications. 
+
+In the medium term, we will transition to consuming this data via versioned APIs from the respective applications once they become available.
+
+## Reasons for the Decision
+
+1. **Access to Current Data**: Leveraging the ADB ensures we have access to data that is synced daily from the relevant applications, enabling us to work with up-to-date information.
+
+2. **Simplicity and Feasibility**: This approach avoids dependencies on external teams delivering APIs within our tight timelines, reducing implementation risk.
+
+3. **Performance**: Consolidating data into a single "super atomic" call from the ADB ensures our application remains performant, while allowing us to query the exact data we need.
+
+4. **Maintainability**: While interim integration requires managing changes to ADB schema, transitioning to APIs later will improve maintainability with version control and documentation inherent to API usage.
+
+## Consequences
+
+### Short-Term Benefits
+- **Rapid Development**: We can meet project deadlines without waiting for API availability.
+- **Centralised Data Access**: ADB provides a unified source of truth for data from multiple applications.
+
+### Drawbacks
+1. **Dependence on ADB Schema**: Changes in the data model or structure of contributing applications may impact our queries and require maintenance.
+2. **Limited Data Validation**: Relying on ADB means less control over the accuracy of the data, as it is aggregated and not fetched directly from source systems.
+
+### Medium-Term Considerations
+- Transitioning to APIs will provide improved control, validation, and resilience against schema changes.
+
+## Trade-offs
+
+### Simplified Implementation vs. Future Maintenance
+This decision simplifies our immediate implementation but introduces potential maintenance overhead as ADB schemas evolve with changes in upstream applications.
+
+### Interim vs. Long-Term Solutions
+Our interim reliance on ADB data ensures we can deliver the feature on time but will require effort to migrate to API-based integration later.
+
+## Requirements for API Integration (Medium-Term)
```

Comment:
I would add that if we are consuming API data then we need those APIs to be able to provide data providence as well, so an inclusion of a last updated or data source field would be important